integrated_channels/sap_success_factors/utils.py

```diff
@@ -93,3 +239,27 @@ def current_time_is_in_interval(start, end):
     interval_start = parse_lms_api_datetime(start or UNIX_MIN_DATE_STRING)
     interval_end = parse_lms_api_datetime(end or UNIX_MAX_DATE_STRING)
     return interval_start <= timezone.now() <= interval_end
+
+
+def parse_language_code(code):
+    """
+    Transform an ISO language code (for example, en-us) into a language name
+    in the format read by SuccessFactors.
+    """
+    if code is None:
+        return 'English'
+    components = code.split('-')
+    language_code = components[0]
+    if 3 > len(components) > 1:
```

Comment:
I'm trying to understand why we would not just say "2" here -- must be missing something 😄